.github/workflows/pr-validation.yml

```diff
@@ -0,0 +1,36 @@
+name: PR Validation
+on:
+  push:
+    branches-ignore:
+      - master
```

Comment:
I am not sure if excluding `master` from this workflow is a good idea, does anyone have an opinion?

Reply:
We didn't exclude it for `scala-http-client`, so I'd do the same here.

Reply:
Done in 2f75e7f